mite_selenium/__init__.py

```diff
@@ -32,8 +33,8 @@ def __init__(self, context):
         self._proxy = self._spec_import_if_not_none("webdriver_proxy")
         self._browser_profile = self._spec_import_if_not_none("webdriver_browser_profile")
         self._options = self._spec_import_if_not_none("webdriver_options")
-        self._options.headless = self._context.config.get("webdriver_options_headless", True)
-
+        if self._options:
+            self._options.headless = self._context.config.get("webdriver_options_headless", True)
```

Comment:
If `self._options` has a value, that's because it was set to something by the user.  Since the user has control of the object whose value is assigned to `self._options`, the user can just set headless on that object.  So we don't need a second user option (`webdriver_options_headless`) for that case.

OTOH, I do see the value of having `webdriver_options_headless` in the case where the user is not setting the options to anything.  It would be convenient not to need to create a whole `Options` object (imported from selenium lib) just to set headless, but rather be able to set it via a simple boolean.  We don't _have_ to offer this flexibility, but we could.

If you accept this analysis, then the conditional here (`if self._options`) has the wrong polarity (we should only be paying attention to `webdriver_options_headless` in the case `if not self.options`)